server/models/db/drop-tables.js

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require('pg');
+
+
+const config = require('../../../config')
+
+const DATABASE_URL = config.appConfig.DATABASE_URL
```

Comment:
this URL is missing in your config.js file.

Reply:
Covered.